src/main/java/com/locallampoon/fiveh/core/Player.java

```diff
@@ -170,6 +170,15 @@ private void changeInventorySize() {
 
     void attack(Monster monster) {
         int damage = getStrength();
+        for (String item : this.getInventory()) {
+            if (monster.getWeaknesses().contains(item)) {
+                if (narrativePanel != null) {
+                    narrativePanel.appendTextArea("You used " + item + "!");
+                }
+                damage += 5;
+                break;
+            }
+        }
```

Comment:
So instead of using a specific item to attack, we check all inventory items. Got you. That means if the player carries the monster's weakness items, it will cause extra damage. I will take a look at how we control Room's inventory. :)

Reply:
When our game gets bigger, the player can have multiple items and kill the monster in one shot :)